packages/nextjs/src/config/index.ts

```diff
@@ -17,13 +18,21 @@ export function withSentryConfig(
   if (typeof userNextConfig === 'function') {
     return function(phase: string, defaults: { defaultConfig: NextConfigObject }): NextConfigObject {
       const materializedUserNextConfig = userNextConfig(phase, defaults);
+      const sentryWebpackPluginOptionsWithSources = includeDistDir(
```

Comment:
Can we make it so that this is generalizable for properties beyond dist dir? I’d rather us approach this generically right now, instead having to refactor in the future when we try to support another option.